researcher_ids/RI_Claims_V1.md

```diff
@@ -839,8 +844,8 @@ Types](#custom-passport-visa-types) and still be compliant.
 
 -   The identity as indicated by the {"iss", "sub"} pair (aka "[Passport Visa
```

Comment:
Is it possible to have a more realistic example here (not `sub1`)?

Reply:
done